xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-test/xwiki-platform-like-test-pageobjects/src/main/java/org/xwiki/like/test/po/UserProfileLikePagesPage.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.like.test.po;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.xwiki.livedata.test.po.LiveDataElement;
+import org.xwiki.model.reference.DocumentReference;
+import org.xwiki.test.ui.po.BaseElement;
+
+/**
+ * Provides the operations to interact with the Like Pages tab of a user profile.
+ *
+ * @version $Id$
+ * @since 13.4RC1
+ */
+public class UserProfileLikePagesPage extends BaseElement
+{
+    /**
+     * The Title column name.
+     */
+    public static final String TITLE_COLUMN_NAME = "Title";
+
+    /**
+     * The Location column name.
+     */
+    public static final String LOCATION_COLUMN_NAME = "Location";
+
+    /**
+     * The Likes column name.
+     */
+    public static final String LIKES_COLUMN_NAME = "Likes";
+
+    private final String userName;
+
+    /**
+     * Default constructor.
+     *
+     * @param userName a user name
```

Comment:
Doesn't feel useful to me...

Reply:
See https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HJavadocBestPractices

Reply:
what is a "user name"? Is it "Vincent" (for ex) or "XWiki.Vincent"? At least you should explain that and always give an example.